portal/server/helpers/getFormattedReportPeriodSpan.ts

```diff
@@ -0,0 +1,45 @@
+import { format } from 'date-fns';
+import { ReportPeriod } from '../types/utilisation-reports';
+
+/**
+ * Gets the formatted report period span
+ * @param reportPeriod - The report period
+ * @returns The formatted report period span
+ * @example
+ * const reportPeriod = {
+ *   start: { month: 1, year: 2024 },
+ *   end: { month: 1, year: 2024 },
+ * };
+ *
+ * const formattedReportPeriodSpan = getFormattedReportPeriodSpan(reportPeriod);
+ * console.log(formattedReportPeriodSpan); // January 2024
+ * @example
+ * const reportPeriod = {
+ *   start: { month: 1, year: 2024 },
+ *   end: { month: 2, year: 2024 },
+ * };
+ *
+ * const formattedReportPeriodSpan = getFormattedReportPeriodSpan(reportPeriod);
+ * console.log(formattedReportPeriodSpan); // January to February 2024
+ * @example
+ * const reportPeriod = {
+ *   start: { month: 12, year: 2023 },
+ *   end: { month: 1, year: 2024 },
+ * };
+ *
+ * const formattedReportPeriodSpan = getFormattedReportPeriodSpan(reportPeriod);
+ * console.log(formattedReportPeriodSpan); // December 2023 to January 2024
+ */
+export const getFormattedReportPeriodSpan = (reportPeriod: ReportPeriod): string => {
```

Comment:
I've thought a bit more about naming this and I think `period` and `span` mean the same thing, essentially `the length of time` so I think we can get away with `getFormattedReportPeriod` but understand if you think that's harder for someone new to follow